Makefile

```diff
@@ -47,14 +47,15 @@ start:
 dev-server: CONFIG_ENV = development
 dev-server: CALYPSO_ENV = desktop-development
 dev-server: NODE_ENV = development
-dev-server: checks desktop$/config.json
+dev-server: checks desktop$/merged-config-development.json
 	@echo "\n\n$(GREEN)+------------------------------------------------+"
 	@echo "|                                                |"
 	@echo "|    Wait for calypso to start the dev server    |"
 	@echo "|       and start the app with \`make dev\`        |"
 	@echo "|                                                |"
 	@echo "+------------------------------------------------+$(RESET)\n\n"
 
+	cp desktop$/merged-config-$(CONFIG_ENV).json desktop$/config.json
```

Comment:
Copying this manually as part of this target is still somewhat suboptimal (but a consequence of not being able to pass around target variables).